File: week_10/c_sharp_course/week_1/day_4/composition/starter_code/wizard_management_specs/DragonTest.cs
using NUnit.Framework;

namespace WizardManagement { 

  [TestFixture]
  public class DragonTest {
    Dragon dragon;

    [SetUp]
    public void Init() 
    {
      dragon = new Dragon("Bob");
    }

    [Test]
    public void HasName()
    {
      Assert.AreEqual("Bob", dragon.Name);
    }

    [Test]
    public void CanFly()
    {
      Assert.AreEqual("Standing up tall, beating wings, lift off!", dragon.Fly());
    }
  }
}

File: week_06/day_3/composition/Composition_Start/app/src/test/java/example/codeclan/com/composition_start/DragonTest.java
package example.codeclan.com.composition_start;

/**
 * Created by user on 28/08/2017.
 */

import static org.junit.Assert.*;
import org.junit.*;

import example.codeclan.com.composition_start.wizard_management.*;


public class DragonTest {

    Dragon dragon;

    @Before
    public void before(){
        dragon = new Dragon("Bob");
    }

    @Test
    public void hasName(){
        assertEquals("Bob", dragon.getName());
    }

    @Test
    public void canFly(){
        assertEquals("Standing up tall, beating wings, lift off!", dragon.fly());
    }
}
